code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -68,6 +68,9 @@
 		user_unbuckle_mob(buckled_mobs[1], user)
 	else
 		if(user.combat_mode)
+			//Return here as the tippable component will do the work instead.
+			if(LAZYACCESS(modifiers, RIGHT_CLICK))
```

Comment:
use `attack_hand_secondary` and change `tippable` to use a signal that `attack_hand_secondary` fires PLEASE

Reply:
OK MAN OK